src/Tizen.NUI/src/devel/Lite/L.Corner.cs

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright(c) 2025 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+
+namespace Tizen.NUI.L
+{
+    /// <summary>
+    /// Defines a value type of corner radius.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    [StructLayout(LayoutKind.Sequential)]
+    public struct Corner
+    {
+        public static readonly Corner Default = new Corner(-1, -1, -1, -1);
+        public static readonly Corner Zero = new Corner(0.0f, 0.0f, 0.0f, 0.0f);
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Corner"/> struct.
+        /// </summary>
+        /// <param name="topLeft">The top-left value.</param>
+        /// <param name="topRight">The top-right value.</param>
+        /// <param name="bottomRight">The bottom-right value.</param>
+        /// <param name="bottomLeft">The bottom-left value.</param>
+        public Corner(float topLeft, float topRight, float bottomRight, float bottomLeft)
+        {
+            TopLeft = topLeft;
+            TopRight = topRight;
+            BottomRight = bottomRight;
+            BottomLeft = bottomLeft;
+        }
+
+        public readonly bool IsDefault => TopLeft == -1 && TopRight == -1 && BottomRight == -1 && BottomLeft == -1;
+
+        public readonly bool IsZero => TopLeft == 0 && TopRight == 0 && BottomRight == 0 && BottomLeft == 0;
+
+        /// <summary>
+        /// The radius of the top left corner of the rectangle.
+        /// </summary>
+        public float TopLeft { get; }
+
+        /// <summary>
+        /// The radius of the top right corner of the rectangle.
+        /// </summary>
+        public float TopRight { get; }
+
+        /// <summary>
+        /// The radius of the bottom left corner of the rectangle.
+        /// </summary>
+        public float BottomLeft { get; }
+
+        /// <summary>
+        /// The radius of the bottom right corner of the rectangle.
+        /// </summary>
+        public float BottomRight { get; }
```

Comment:
```suggestion
        /// <summary>
        /// The radius of the bottom right corner of the rectangle.
        /// </summary>
        public float BottomRight { get; }

        /// <summary>
        /// The radius of the bottom left corner of the rectangle.
        /// </summary>
        public float BottomLeft { get; }
```

The order of property define is important! (Since it is `LayoutKind.Sequential`)

Reply:
Good point. It's a big mistake! Even though I decided not to use sequential thing, I fixed it to align with constructor parameter sequence.